### ISSUE TITLE
Filter out Eloqua's email tracker from URLs

### ISSUE THREAD

**Issue by fmarier** [brave/brave-browser#10151]
Emails sent via the Eloqua platform add two high-entropy tracking query string parameters to all outbound links:
- [ `elq`](https://docs.oracle.com/en/cloud/saas/marketing/eloqua-user/Help/EloquaAsynchronousTrackingScripts/EloquaTrackingParameters.htm#Recipien): tracks individual email addresses
- [`elqTrackId`](https://docs.oracle.com/en/cloud/saas/marketing/eloqua-user/Help/EloquaAsynchronousTrackingScripts/EloquaTrackingParameters.htm#ElqTrack): tracks individual links inside emails

Here's an example link cached by Google which includes both of these parameters:

    https://www.aon.com/attachments/human-capital-consulting/2017-hot-topics-financialwellbeing-report-final-january.pdf?elqTrackId=284B8F7AF66B224DAF7C6CA203C2907B&elq=513883ed34ea4b718024d1f1648855f7&elqaid=25542&elqat=1&elqCampaignId=

Sadly, the `elq` param is needed to prefill the email address on the unsubscribe page. The `elqTrackId` param is not needed for the unsubscribe link to work though.

The same parameters are used for the script-less [tracking pixel](https://docs.oracle.com/en/cloud/saas/marketing/eloqua-user/Help/EloquaAsynchronousTrackingScripts/Tasks/TrackingImageTag.htm) which follows this pattern:

    http://*.t.eloqua.com/visitor/v200/svrGP

Here's the relevant entry in EasyPrivacy:

    ||eloqua.com^$~stylesheet,third-party

More details can be found in [Oracle's tracking script documentation](https://docs.oracle.com/en/cloud/saas/marketing/eloqua-user/pdf/AsynchronousVisitorTrackingScripts.pdf).

**Comment by fmarier** [comment]
@ryanbr Does this rule cover tracking images too?

    ||eloqua.com^$~stylesheet,third-party

**Comment by ryanbr** [comment]
Yes, that will work. embedded tracking images/pixels. @fmarier Just not click through trackers

**Comment by fmarier** [comment]
@ryanbr Ok thanks, so I guess it reads "stylesheets, and all third-party requests", not "third-party stylesheet requests". I pulled this rule out of EasyPrivacy.

**Comment by arthuredelstein** [comment]
One possible option here is to retain the `elq` parameter if the URL also contains the string "unsubscribe". This seems to be a common pattern in practice.

**Comment by bsclifton** [comment]
cc: @fmarier for triage

**Comment by fmarier** [comment]
It also causes webcompat issues: https://github.com/AdguardTeam/AdguardFilters/issues/230641

So I think we should simply omit this one.
